serverless.yml

```diff
@@ -67,4 +67,47 @@ custom:
     excludeFiles: src/**/*.test.js # Provide a glob for files to ignore
   # https://www.serverless.com/plugins/serverless-offline#usage-with-webpack - See Usage and command line options
   serverless-offline:
-    httpPort: ${env:AWS_SERVER_PORT, 3001}
+    httpPort: ${env:AWS_SERVER_PORT, 3020}
+  dynamodb:
+    start:
+      port: ${env:DYNAMO_PORT, 8020}
+      inMemory: true
+      migrate: false
+      seed: false
+      noStart: true
+    seed:
+      trailer-registration:
+        sources:
+          - table: cvs-local-trailer-registration 
+            sources: [./tests/resources/trailer-registration.json]
```

Comment:
@JonLazariniDD  is the intention to keep test data under ```test/resources``` or ```/data```

Reply:
moved under data.